containers/daap-python-base/src/var/task/versioning.py

```diff
@@ -141,6 +175,61 @@ def metadata_update_type(data_product_metadata) -> UpdateType:
         return UpdateType.MinorUpdate
 
 
+def schema_update_type(
```

Comment:
Can you add more unit tests to cover the various scenarios this function handles?